src/configurations/cancercomplexity/routesConfig.ts

```diff
@@ -30,13 +30,16 @@ const routes: GenericRoute[] = [
       },
```

Comment:
Some of the components in CancerComplexity had their titles embedded in their custom components.  Now it sets the title in the standard way (to support the Home menu dropdown, and make them all consistent)